src/routers/secure/items.py

```
@@ -400,6 +400,8 @@ async def get_item(
             else:
                 raise HTTPException(status_code=404, detail="Item not found")
         except Exception as e:
+            if isinstance(e, HTTPException):
+                raise e
```

Ruff (0.14.3)

Use `raise` without specifying exception name

Remove exception name

(TRY201)